hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/s3guard.md

```diff
@@ -181,7 +182,7 @@ removed on `S3AFileSystem` level.
 
 ```xml
 <property>
-    <name>fs.s3a.metadatastore.authoritative.dir.ttl</name>
+    <name>fs.s3a.metadatastore.metadata.ttl</name>
```

Comment:
with a switch to `Configuration.getTimeDuration()` then a value like `15m` or `1h` will work here and in any core-default value